build/js/live-editor.output_pjs.js

```diff
@@ -2515,7 +2520,6 @@ window.PJSOutput = Backbone.View.extend({
 
     kill: function() {
         this.canvas.exit();
-        this.canvas.hide();
```

Comment:
canvas doesn't implement this method.  This was found before because "kill" was never being called.